### PR TITLE
Store last language used in application settings

### PR DESCRIPTION
**Pull request by occivink**
Follow-up to the previous PR. The last language used is now also stored in settings

### CODE REVIEW

File: studio/src/window.cpp [hunk 1]
```diff
@@ -46,10 +46,40 @@ namespace Studio {
...
+        if (language_str == "guile")
+            language = Language::LANGUAGE_GUILE;
+        else if (language_str == "python")
+            language = Language::LANGUAGE_PYTHON;
+    }
+    // otherwise the editor will pick the default
+
+    editor = new Editor(language);
+
+    auto storeLanguage = [this] {
```
Comment (mkeeter) [inline studio/src/window.cpp:72]: Does this _need_ to be a lambda function?  If not, I'd prefer a normal slot in the `Window` class, to minimize cleverness.

Comment (occivink) [reply studio/src/window.cpp]: It does not need to, just personal preference. My reasoning is that this way, all code related to reading/writing language to settings is localized to these 30 lines consecutive lines. 
But I understand your point of view as well, so I've changed it

**Comment by mkeeter** [comment]
Looks good, merged.

Also, 🎉 congrats 🎉  – you've now got commit privileges to the repo, since you seem interested in improving the UI!

Feel free to keep opening PRs for changes that require feedback, but for small changes / bug fixes, you're welcome to push your own commits.

**Comment by occivink** [comment]
thank you, this is an honor. Though my contributions have been mostly to scratch some own itches, my hope is that they can benefit libfive as well

**Pull request merged**